engines/hdb/hdb.cpp

```diff
@@ -167,6 +179,12 @@ bool HDBGame::init() {
 	_lua->init();
 	_menu->init();
 
+	// Query the selected music device (defaults to MT_AUTO device).
+	Common::String selDevStr = ConfMan.hasKey("music_driver") ? ConfMan.get("music_driver") : Common::String("auto");
+	MidiDriver::DeviceHandle dev = MidiDriver::getDeviceHandle(selDevStr.empty() ? Common::String("auto") : selDevStr);
```

Comment:
You don't need to check if selDevStr is empty, you've done that right above

```suggestion
	MidiDriver::DeviceHandle dev = MidiDriver::getDeviceHandle(selDevStr);
```

Reply:
I am unsure why I thought that this was needed (I did it that way for Blade Runner). Maybe it was leftover, or just making sure -- I suppose it's possible for the "music_driver" key to exist and still be an empty string, especially since we manual editing of ScummVM ini is possible, but that is a very low chance.

I'll push a commit without the check.